### PR TITLE
feat(plasma-tokens): Add tokens generate fot ios-swift platform and update generator

### PR DESCRIPTION
**Pull request by neretin-trike**
:baby_chick: Download canary assets:

[colors--canary.226.3764765981.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.226.3764765981.xml)  
[color_metro_ios-swift--canary.226.3764765981.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_ios-swift--canary.226.3764765981.swift)  
[color_metro_kotlin--canary.226.3764765981.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_kotlin--canary.226.3764765981.kt)  
[color_metro_react-native--canary.226.3764765981.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_react-native--canary.226.3764765981.ts)  
[color_sbermarket_ios-swift--canary.226.3764765981.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.226.3764765981.swift)  
[color_sbermarket_kotlin--canary.226.3764765981.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.226.3764765981.kt)  
[color_sbermarket_react-native--canary.226.3764765981.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.226.3764765981.ts)  
[color_sberprime_ios-swift--canary.226.3764765981.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.226.3764765981.swift)  
[color_sberprime_kotlin--canary.226.3764765981.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.226.3764765981.kt)  
[color_sberprime_react-native--canary.226.3764765981.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.226.3764765981.ts)  
[color_selgros_ios-swift--canary.226.3764765981.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_ios-swift--canary.226.3764765981.swift)  
[color_selgros_kotlin--canary.226.3764765981.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_kotlin--canary.226.3764765981.kt)  
[color_selgros_react-native--canary.226.3764765981.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_react-native--canary.226.3764765981.ts)  
[color_smbusiness_ios-swift--canary.226.3764765981.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_ios-swift--canary.226.3764765981.swift)  
[color_smbusiness_kotlin--canary.226.3764765981.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_kotlin--canary.226.3764765981.kt)  
[color_smbusiness_react-native--canary.226.3764765981.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_react-native--canary.226.3764765981.ts)  
[PlasmaTokensColor--canary.226.3764765981.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.226.3764765981.swift)  
[typo_mage_ios-swift--canary.226.3764765981.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.226.3764765981.swift)  
[typo_mage_kotlin--canary.226.3764765981.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.226.3764765981.kt)  
[typo_mage_react-native--canary.226.3764765981.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.226.3764765981.ts)  
[typo_plasma_ios-swift--canary.226.3764765981.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.226.3764765981.swift)  
[typo_plasma_kotlin--canary.226.3764765981.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.226.3764765981.kt)  
[typo_plasma_react-native--canary.226.3764765981.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.226.3764765981.ts)  
[typo_ruler_ios-swift--canary.226.3764765981.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.226.3764765981.swift)  
[typo_ruler_kotlin--canary.226.3764765981.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.226.3764765981.kt)  
[typo_ruler_react-native--canary.226.3764765981.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.226.3764765981.ts)  
[typo_sage_ios-swift--canary.226.3764765981.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.226.3764765981.swift)  
[typo_sage_kotlin--canary.226.3764765981.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.226.3764765981.kt)  
[typo_sage_react-native--canary.226.3764765981.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.226.3764765981.ts)  
[typo_sbermarket_ios-swift--canary.226.3764765981.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.226.3764765981.swift)  
[typo_sbermarket_kotlin--canary.226.3764765981.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.226.3764765981.kt)  
[typo_sbermarket_react-native--canary.226.3764765981.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.226.3764765981.ts)  
[typo_soulmate_ios-swift--canary.226.3764765981.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.226.3764765981.swift)  
[typo_soulmate_kotlin--canary.226.3764765981.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.226.3764765981.kt)  
[typo_soulmate_react-native--canary.226.3764765981.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.226.3764765981.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.123.0-canary.226.3764765981.0
  npm install @salutejs/plasma-core@1.88.0-canary.226.3764765981.0
  npm install @salutejs/plasma-icons@1.115.0-canary.226.3764765981.0
  npm install @salutejs/plasma-temple@1.120.0-canary.226.3764765981.0
  npm install @salutejs/plasma-tokens@1.32.0-canary.226.3764765981.0
  npm install @salutejs/plasma-typo@0.29.0-canary.226.3764765981.0
  npm install @salutejs/plasma-ui@1.153.0-canary.226.3764765981.0
  npm install @salutejs/plasma-web@1.150.0-canary.226.3764765981.0
  npm install @salutejs/plasma-cy-utils@0.38.0-canary.226.3764765981.0
  npm install @salutejs/plasma-sb-utils@0.86.0-canary.226.3764765981.0
  npm install @salutejs/plasma-tokens-android@2.39.0-canary.226.3764765981.0
  npm install @salutejs/plasma-tokens-ios-swift@2.39.0-canary.226.3764765981.0
  # or 
  yarn add @salutejs/plasma-b2c@1.123.0-canary.226.3764765981.0
  yarn add @salutejs/plasma-core@1.88.0-canary.226.3764765981.0
  yarn add @salutejs/plasma-icons@1.115.0-canary.226.3764765981.0
  yarn add @salutejs/plasma-temple@1.120.0-canary.226.3764765981.0
  yarn add @salutejs/plasma-tokens@1.32.0-canary.226.3764765981.0
  yarn add @salutejs/plasma-typo@0.29.0-canary.226.3764765981.0
  yarn add @salutejs/plasma-ui@1.153.0-canary.226.3764765981.0
  yarn add @salutejs/plasma-web@1.150.0-canary.226.3764765981.0
  yarn add @salutejs/plasma-cy-utils@0.38.0-canary.226.3764765981.0
  yarn add @salutejs/plasma-sb-utils@0.86.0-canary.226.3764765981.0
  yarn add @salutejs/plasma-tokens-android@2.39.0-canary.226.3764765981.0
  yarn add @salutejs/plasma-tokens-ios-swift@2.39.0-canary.226.3764765981.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-226/
  <details>
    <summary>New documentation</summary>
    <br />

    http://plasma.fine.sberdevices.ru/pr/demo-canvas-app-pr-226/

  </details>

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-226/
  ui storybook: http://plasma.sberdevices.ru/pr-226/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-226/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-226/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-226/temple-storybook/
  <details>
    <summary>New documentation</summary>
    <br />

    website: http://plasma.fine.sberdevices.ru/pr/pr-226/
    ui storybook: http://plasma.fine.sberdevices.ru/pr/pr-226/ui-storybook/
    web storybook: http://plasma.fine.sberdevices.ru/pr/pr-226/web-storybook/
    b2c storybook: http://plasma.fine.sberdevices.ru/pr/pr-226/b2c-storybook/
    temple storybook: http://plasma.fine.sberdevices.ru/pr/pr-226/temple-storybook/

  </details>

**Review by Yakutoc — APPROVED** [review]

File: packages/plasma-typo/src/archetypes/sbermarket.ts [hunk 1]
```diff
@@ -39,7 +39,7 @@ export const baseTypoS: TypoProps = {
     },
     'h1-bold': {
         'font-size': '2rem',
-        'font-weight': '6700',
+        'font-weight': '700',
```
Comment (Yeti-or) [inline packages/plasma-typo/src/archetypes/sbermarket.ts:42]: опечатка была ?

**Review by Yeti-or — APPROVED** [review]

File: packages/plasma-typo/src/archetypes/sbermarket.ts (hunk 1 above)
Comment (neretin-trike) [reply packages/plasma-typo/src/archetypes/sbermarket.ts:42]: ага :с

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-226/
  <details>
    <summary>New documentation</summary>
    <br />

    http://plasma.fine.sberdevices.ru/pr/demo-canvas-app-pr-226/

  </details>

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-226/
  ui storybook: http://plasma.sberdevices.ru/pr-226/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-226/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-226/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-226/temple-storybook/
  <details>
    <summary>New documentation</summary>
    <br />

    website: http://plasma.fine.sberdevices.ru/pr/pr-226/
    ui storybook: http://plasma.fine.sberdevices.ru/pr/pr-226/ui-storybook/
    web storybook: http://plasma.fine.sberdevices.ru/pr/pr-226/web-storybook/
    b2c storybook: http://plasma.fine.sberdevices.ru/pr/pr-226/b2c-storybook/
    temple storybook: http://plasma.fine.sberdevices.ru/pr/pr-226/temple-storybook/

  </details>

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-226/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-226/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-226/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-226/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-226/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-226/temple-storybook/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-226/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-226/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-226/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-226/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-226/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-226/temple-storybook/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-226/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-226/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-226/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-226/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-226/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-226/temple-storybook/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-226/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-226/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-226/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-226/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-226/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-226/temple-storybook/

File: packages/plasma-typo/src/helpers.ts [hunk 2]
```diff
@@ -134,8 +134,13 @@ export const createVariablesByArcheType = ({
...
+                ${
+                    typoM &&
+                    css`
+                        @media (min-width: ${560 * deviceScale}px) and (max-width: ${959 * deviceScale}px) {
+                            ${prepareStandardBreakpointTypo(typoM)}
+                        }
+                    `
                 }
 
                 @media (min-width: ${960 * deviceScale}px) {
```
Comment (Yeti-or) [inline packages/plasma-typo/src/helpers.ts:146]: а не надо для остальных также как typoM ?

File: packages/plasma-tokens/lib/generator/formatters/typoKotlin.ts [hunk 3]
```diff
@@ -18,63 +28,90 @@ const getKotlinTemplate = (
...
 import androidx.compose.ui.text.font.FontFamily
-import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.text.style.TextDecoration
-import androidx.compose.ui.unit.TextUnitType
 import androidx.compose.ui.unit.sp
-import androidx.compose.ui.platform.LocalDensity
-import androidx.compose.runtime.Composable
+import ru.sbermarket.sm_theme.R
```
[the remark this answers is not in the document]
Comment (neretin-trike) [reply packages/plasma-tokens/lib/generator/formatters/typoKotlin.ts:40]: пока не придумал, как это можно универсально сделать на данный моммент. В будущем скорее всего мы сможем это разруливать какими-нибудь env'ами во время CI для разных продуктов. А сейчас предлагаю такой импорт оставить

File: packages/plasma-typo/src/helpers.ts (hunk 2 above)
Comment (neretin-trike) [reply packages/plasma-typo/src/helpers.ts:146]: да я вот думал, надо ли. Если тебе тоже показалось так, то давай добавлю

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-226/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-226/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-226/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-226/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-226/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-226/temple-storybook/

**Pull request merged**